docusaurus/docs/develop/guard/walkthrough.md

```diff
@@ -0,0 +1,577 @@
+---
```

Comment:
@commoddity Are there any net-new changes here or is it a copy-paste of the previous docs?

Skipping reviewing this for now.

Reply:
This was mistakenly added from the docs branch I'm working on. I've removed it.